avro_schema/backend.lua

```diff
@@ -591,22 +593,42 @@ emit_nested_block = function(ctx, block, cc, res)
     insert(queue, cc)
 end
 
-local locals_tab = {
-    [0] = 'local x%d',
-    'local x%d, x%d',
-    'local x%d, x%d, x%d'
-}
+-- The procedure emits local variables with names corresponding to their
+-- register numbers in `ir` representation
+local function emit_locals(varmap, nservice_fields, res)
+    if nservice_fields then
+        -- Allocate place for service_fields starting from 1
+        -- local sf1
```

Comment:
Redundant comment.

Reply:
It is how service field variables are emitted. Ok, I will reword it.